core/rest-cxf/src/main/java/org/apache/syncope/core/rest/cxf/service/ReconciliationServiceImpl.java

```diff
@@ -98,7 +98,8 @@ public ReconStatus status(final ReconQuery query) {
         validate(query);
 
         if (query.getAnyKey() != null) {
-            return logic.status(query.getAnyTypeKey(), query.getResourceKey(), query.getAnyKey());
+            return logic.status(query.getAnyTypeKey(), query.getResourceKey(), query.getAnyKey(),
+                    query.getMoreAttrsToGet());
```

Comment:
`query,getMoreAttrsToGet()` can be `null` (see elsewhere methods in this class)

Reply:
You can for instance replace

```query.getMoreAttrsToGet()```

with

```java
Optional.ofNullable(query.getMoreAttrsToGet()).orElse(new HashSet<>())
```

Reply:
You're right, my bad. I'm going to fix